myapp/app/views/tasks/edit.html.erb

```diff
@@ -1,3 +1,3 @@
-<h1>Edit Task</h1>
+<h1><%= t '.title' %></h1>
```

Comment:
【FYI】
In Rakuma, a member uses this helper method by writing `i18n.t`!
`i18n` can be omitted, but we want to indicate explicitly that we are using i18n.

Reply:
Thank you for telling me. That's good.
I'll use `I18n.t`